opentelemetry-api/tests/propagators/test_propagators.py

```diff
@@ -49,6 +49,18 @@ def test_propagators(propagators):
 
         reload(opentelemetry.propagate)
 
+    @patch.dict(environ, {OTEL_PROPAGATORS: "none"})
+    @patch("opentelemetry.propagators.composite.CompositePropagator")
+    def test_no_propagators_loaded(self, mock_compositehttppropagator):
```

Comment:
This test doesn't fail against main branch

Reply:
@emdneto I apologize for the confusion. Could you clarify what you would like me to address?

Reply:
@Annosha The should fail on main and raise and exception (as reported in the issue). If it doesn't fail with the current code it means it's not a good test to evaluate if the fix is working.

Reply:
@xrmx and @emdneto The test is failing after fixes in testing logic. This failure confirms that the test accurately detects the issue. Here is the test log on my machine:
![test-fail-for-PR-OTEL-Propagators-NONE-vlaue](https://github.com/user-attachments/assets/d1a2f16a-45f7-407c-93db-e1ba1a35084e)
Could you please review this on your end and advise on any further improvements needed?